### PR TITLE
Undefine requestIdleCallback in the Nuxt test for consistency.

### PR DESCRIPTION
**Pull request by rniwa**
We can undo this change once Safari / WebKit adds the support for requestIdleCallback.

### CODE REVIEW

**Comment by rniwa** [comment]
See https://github.com/WebKit/Speedometer/issues/240

**Comment by rniwa** [comment]
http://localhost:8000/?suites=NewsSite-Nuxt#summary

**Chrome**
* Before:
    * 18.3 ± 0.61
    * 18.5 ± 0.29
    * 18.5 ± 0.26

* After:
    * 18.9 ± 0.49
    * 18.8 ± 0.43
    * 18.7 ± 0.36

**Firefox**
* Before:
    * 15.3 ± 0.92
    * 14.6 ± 0.53
    * 14.7 ± 0.80

* After:
    * 14.9 ± 0.86
    * 14.7 ± 0.71
    * 15.0 ± 0.68


**Safari**
* Before:
    * 12.7 ± 0.32
    * 12.1 ± 0.41
    * 12.8 ± 0.31

* After:
    * 12.7 ± 0.45
    * 12.5 ± 0.34
    * 12.1 ± 0.50

[nuxt-idle-callback.zip](https://github.com/WebKit/Speedometer/files/11777271/nuxt-idle-callback.zip)

File: resources/newssite/news-nuxt/dist/index.html [hunk 1]
```diff
@@ -3,6 +3,6 @@
...
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<meta name="description" content="A news site developed with Nuxt."><link rel="modulepreload" as="script" crossorigin href="./_nuxt/entry.eed2f10a.js"><link rel="preload" as="style" href="./_nuxt/entry.762f5ea3.css"><link rel="prefetch" as="script" crossorigin href="./_nuxt/error-component.a9992003.js"><link rel="stylesheet" href="./_nuxt/entry.762f5ea3.css"></head>
-<body ><div id="__nuxt"></div><script type="application/json" id="__NUXT_DATA__" data-ssr="false">[{"_errors":1,"serverRendered":2,"data":3,"state":4},{},false,{},{}]</script><script>window.__NUXT__={};window.__NUXT__.config={public:{},app:{baseURL:"./",buildAssetsDir:"/_nuxt/",cdnURL:"./"}}</script><script type="module" src="./_nuxt/entry.eed2f10a.js" crossorigin></script></body>
+<meta name="description" content="A news site developed with Nuxt."><link rel="modulepreload" as="script" crossorigin href="./_nuxt/entry.34ceb9be.js"><link rel="preload" as="style" href="./_nuxt/entry.762f5ea3.css"><link rel="prefetch" as="script" crossorigin href="./_nuxt/error-component.004af810.js"><link rel="stylesheet" href="./_nuxt/entry.762f5ea3.css"></head>
```
Comment (bgrins) [inline resources/newssite/news-nuxt/dist/index.html:6]: Unrelated to this change, but I seem to remember some issues with modulepreload causing network traffic during measured time in the editors workload https://github.com/WebKit/Speedometer/pull/81#issuecomment-1462599849. Would have to take a closer look at profiles to see if this is happening here as well.

**Review by bgrins — APPROVED** [review]
Makes sense to me, for consistency

File: resources/newssite/news-nuxt/dist/index.html (hunk 1 above)
Comment (rniwa) [reply resources/newssite/news-nuxt/dist/index.html:6]: Oh, interesting. We should look into that.

**Review by flashdesignory — APPROVED** [review]

**Pull request merged**